pkg/apis/core/types.go

```diff
@@ -4697,7 +4700,10 @@ type SecurityContext struct {
 	// PodSecurityContext, the value specified in SecurityContext takes precedence.
 	// +optional
 	SELinuxOptions *SELinuxOptions
-	// Windows security options.
+	// The Windows specific settings applied to container
+	// if unspecified the options within the container will be used
+	// May also be set in SecurityContext. If set in both SecurityContext and
+	// PodSecurityContext, the value specified in SecurityContext takes precedence
```

Comment:
comment nit: `if unspecified the options within the container will be used May also be set in SecurityContext. If set in both` => `If unspecified, the options in PodSecurityContext will be used. If set in both ...`

Also please add the reference to the feature gates as mentioned above.